src/controllers/index.controller.js

```diff
@@ -189,16 +190,20 @@ function start(req, res) {
   notificationBus.subscribeHathorWallet(req.body['wallet-id'], wallet);
 
   wallet.start().then(info => {
-    console.log(`Wallet started with wallet id ${req.body['wallet-id']}. Full-node info: `, info);
-    initializedWallets[req.body['wallet-id']] = wallet;
+    // The replace avoids Log Injection
+    console.log(
+      `Wallet started with wallet id ${sanitizeLogInput(walletID)}. Full-node info: ${info}`
+    );
+
+    initializedWallets[walletID] = wallet;
```

Comment:
Since this is a `Map`, should we use `initializedWallets.set(walletId, wallet)`?

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-headless/pull/250/commits/f820dd72265bb2085e588e0141e00deb532620df